source/docs/software/using-android-studio.rst

```diff
@@ -262,6 +264,69 @@ Finally, perform a gradle sync.
 Android Debug Bridge
 ====================
 
+Logcat
+------
+The logcat is extremely useful for debugging issues
+with your code at runtime or figuring out what went wrong.
+For example, if your app activity crashes and you pull up
+the log seeing 400 pages of something like this:
+
+.. image:: images/using-android-studio/infinite-recursion.png
+    :alt: Infinite recursion
+
+then, you know that there is infinite recursion somewhere
+in your program.
+
+To use the logcat, plug in your device (or connect via ADB).
+Then, select the app you want to view the logs for. Your window
+should look like this.
+
+.. image:: images/using-android-studio/logcat-window.png
+    :alt: A selected device and app with the error messages
+
+If you have an issue you don't understand, you can print
```

Comment:
Leaving this here will lead to a *lot* of pointless issues on the SDK repo... maybe suggest the FTC discord? :shrug:

Reply:
Ah good point. Sometimes issues can be resolved just by having someone who understands error messages read it. Good call, I'll change that.